### ISSUE TITLE
Games do not launch, regardless of settings.

### ISSUE THREAD

**Issue by TheGershon** [Vita3K/Vita3K#2421]
Games immediately hang upon launching. Have tried changing GPU backend, upscaling setting, CPU backend, Audio backend, different games, all for the log to endlessly repeat the same error. I'm running on a Ryzen 7 3700X & an RTX 2070 SUPER on Windows 10.
I've noticed that it starts with one error before switching to a different one:

![image](https://user-images.githubusercontent.com/6880780/224953028-0c950c23-f19d-41e3-960a-01013165267c.png)

![image](https://user-images.githubusercontent.com/6880780/224952565-65660e16-a30b-4f73-86cd-9a46a4511e6b.png)

[vita3k.log](https://github.com/Vita3K/Vita3K/files/10966474/vita3k.log)

**Comment by nishinji** [comment]
Does it happen in all games?
It looks like eboot.bin is bad

**Comment by TheGershon** [comment]
Yeah, I'll give redumping a shot and see if that changes anything, although it'd be weird for something to have gone wrong twice

**Comment by scribam** [comment]
@TheGershon which game is it?

**Comment by TheGershon** [comment]
I genuinely don't remember how I resolved this, but it's been 3 years. Closing.
